eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -44,6 +44,9 @@ parameters:
   - name: UsePlatformContainer
     type: boolean
     default: false
+  - name: VcpkgCacheMode
+    type: string
+    default: 'read'
```

Comment:
from where ?

Reply:
from the cache. ( it is set in the vcpkg step, I will add a comment over there for you)